README.rst

```diff
@@ -74,6 +74,40 @@ Setup (integrate into an existing Django project)
         'openwisp_notifications',
      ]
 
+Configure caching (you may use a different cache storage if you want):
+
+.. code-block:: python
+
+    CACHES = {
+        'default': {
+            'BACKEND': 'django_redis.cache.RedisCache',
+            'LOCATION': 'redis://localhost/5',
+            'OPTIONS': {
+                'CLIENT_CLASS': 'django_redis.client.DefaultClient',
+            }
+        }
+    }
+
+    SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
+    SESSION_CACHE_ALIAS = 'default'
+
+Configure celery:
+
+.. code-block:: python
+
+    # here we show how to configure celery with Redis but you can
+    # use other brokers if you want, consult the celery docs
+    CELERY_BROKER_URL = 'redis://localhost/1'
+
+    INSTALLED_APPS.append('djcelery_email')
+    EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
```

Comment:
I don't know why you're mentioning this, it's not needed in what we're doing, I already told you, this is a setting that is performed during deployment and not at application level.

Reply:
I thought this would be helpful for users. Since we deals with emails here, I thought this would be appropriate place to mention this. 

Sorry for the commit message.

Reply:
You may add this suggestion in a different patch, but ensure it's mentioned in a different block of text, eg:

```
# pseudo docs
If you want to configure email sending to happen in the background, do the following:

- pip install ...
- edit settings as follows...
```